pkg/cmd/webhook/webhook_cmd.go

```diff
@@ -45,8 +45,7 @@ func (o *options) setFlags(command *cobra.Command) {
 	command.Flags().StringVar(&o.pipelinesFolderPath, "pipelines-folder", ".", "Folder path to retrieve manifest, eg. /test where manifest exists at /test/pipelines.yaml")
 
 	// access-token option
-	command.Flags().StringVar(&o.accessToken, "access-token", "", "Access token to be used to create Git repository webhook")
-	_ = command.MarkFlagRequired("access-token")
+	command.Flags().StringVar(&o.accessToken, "access-token", "", "Access token to be used to create Git repository webhook. Access token will be updated in the keyring")
```

Comment:
access token is encrypted and stored on local file system by keyring to be reused.